components/Card/Card.tsx

```diff
@@ -62,11 +64,19 @@ export const Card: FC<Props> = ({ template }) => {
 
       <Name>{name}</Name>
       <CollectionName>{collection_name}</CollectionName>
-      {lowestPrice === undefined ? (
-        <ShimmerBlock position="flex-start" />
-      ) : lowestPrice ? (
-        <Price>{lowestPrice}</Price>
-      ) : null}
+      <PriceSection lowestPrice={lowestPrice} type={type} />
     </CardContainer>
   );
 };
+
+const PriceSection = ({ lowestPrice, type }) => {
+  if (type === 'user') {
```

Comment:
why don't you just do (type === 'user' || !lowestPrice) returns null

Reply:
true lul..